doc/release_instructions.md

```diff
@@ -11,20 +11,33 @@ This is because of a dependency on CGO via [this bug](https://github.com/opencon
 You'll need the following credentials / tools:
 
 - goxc (`go get github.com/laher/goxc`)
-- goxc configured ... 
+- goxc configured ...
     - run: `make GITHUB_OAUTH_TOKEN=12345 goxc-generate-local`
     - You can get your personal oauth token here: https://github.com/settings/tokens
 - sha256sum (if you're on mac ... `brew install coreutils`)
 - access to `homebrew-tap` and `www` repositories
-- S3 credentials 
+- S3 credentials
+- A dockerhub account, with write access to https://hub.docker.com/u/pachyderm/
 
 ## Releasing:
 
 1) Make sure your commit has a passing build on travis
 
 2) Update `src/client/version/version.go` version values
 
-3) Release
+3) Run `make doc` with the new version values.
+
+4) At this point, all of our auto-generated documentation should be updated. Push a new commit (to master) with:
+
+```
+> VERSION=<new version, e.g. 1.2.3>
+> git commit -a -m"Update version and run make doc for ${VERSION} point release"
```

Comment:
Nitpick -- If we only use $VERSION in the commit string ... I don't think we need it. If someone fails to set it the message reads 'Update version and run make doc for  point release' which is not good. 

But if we make the example string "Update version and run make doc for VERSION point release" it is at least obvious that they forgot to add it and can run `git --amend`